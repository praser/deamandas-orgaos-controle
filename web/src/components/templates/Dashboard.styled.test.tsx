import React from 'react';
import { render, screen } from '@testing-library/react';
import {
  Topbar,
  MainContent,
  TopControls,
  InlineGroup,
  Content1,
  Content2,
} from './Dashboard.styled';

describe('Dashboard layout styled components', () => {
  const testId = 'layout';

  it('Topbar is expected to match the snapshot', () => {
    render(<Topbar data-testid={testId} />);
    const topbar = screen.getByTestId(testId);
    expect(topbar).toMatchSnapshot();
  });

  it('MainContent is expected to match the snapshot', () => {
    render(<MainContent data-testid={testId} />);
    const mainContent = screen.getByTestId(testId);
    expect(mainContent).toMatchSnapshot();
  });

  it('TopControls is expected to match the snapshot', () => {
    render(<TopControls data-testid={testId} />);
    const topControls = screen.getByTestId(testId);
    expect(topControls).toMatchSnapshot();
  });

  it('InlineGroup is expected to match the snapshot', () => {
    render(<InlineGroup data-testid={testId} />);
    const inlineGroup = screen.getByTestId(testId);
    expect(inlineGroup).toMatchSnapshot();
  });

  it('Content1 is expected to match the snapshot', () => {
    render(<Content1 data-testid={testId} />);
    const content1 = screen.getByTestId(testId);
    expect(content1).toMatchSnapshot();
  });

  it('Content2 is expected to match the snapshot', () => {
    render(<Content2 data-testid={testId} />);
    const content2 = screen.getByTestId(testId);
    expect(content2).toMatchSnapshot();
  });
});
