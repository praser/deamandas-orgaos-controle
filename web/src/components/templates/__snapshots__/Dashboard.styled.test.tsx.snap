// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard layout styled components Content1 is expected to match the snapshot 1`] = `
<div
  class="sc-eCImvq ctukcT"
  data-testid="layout"
/>
`;

exports[`Dashboard layout styled components Content2 is expected to match the snapshot 1`] = `
<div
  class="sc-jRQAMF iKOuxR"
  data-testid="layout"
/>
`;

exports[`Dashboard layout styled components InlineGroup is expected to match the snapshot 1`] = `
<div
  class="sc-hKwCoD xvjnu"
  data-testid="layout"
/>
`;

exports[`Dashboard layout styled components MainContent is expected to match the snapshot 1`] = `
<div
  class="sc-gsDJrp eqafHM"
  data-testid="layout"
/>
`;

exports[`Dashboard layout styled components TopControls is expected to match the snapshot 1`] = `
<div
  class="sc-dkPtyc fOAGCu"
  data-testid="layout"
/>
`;

exports[`Dashboard layout styled components Topbar is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa cIKZCl"
  data-testid="layout"
/>
`;
